spec/support/macros.rb

```diff
@@ -8,4 +8,8 @@ def sign_in(user=nil)
   fill_in "Email Address", with: user.email
   fill_in "Password", with: user.password
   click_button "Sign in"
+end
+
+def click_video(video)
```

Comment:
You don't have to do this right away, but I recommend that you eventually create separate macro files, one for feature specs and one for controller specs.